components/remote_settings/src/client.rs

```diff
@@ -330,6 +330,19 @@ impl GetItemsOptions {
         self
     }
 
+    /// Sets an option to return items whose `field` matches any one of the `values`.
+    ///
+    /// `field` can be a simple or dotted field name, like `author` or
+    /// `author.name`. `values` is a vector of Strings.
+    pub fn eq_any(
```

Comment:
Ah ya, I used `eq_any` because `in` is a keyword, so it's kind of confusing. Also, we already have an `eq` function which does a match to one particular field. So, this is _kind_ of an extension of that: it has to match _any_ of the fields in the list. It is definitely more descriptive than `in`!